app/javascript/components/laboratory/Laboratory.js

```diff
@@ -43,6 +45,7 @@ class Laboratory extends React.Component {
   }
 
   handleDateChange(field, date) {
+    date = formatDate(date);
```

Comment:
I think this introduces a bug that results in an error when you add a Lab Result to a monitoree without a specimen collection date. After doing so, you get this error on the Patient page:
<img width="1012" alt="image" src="https://user-images.githubusercontent.com/8235974/111350424-7b5f0180-8658-11eb-995b-1bb00722a655.png">

 I haven't stepped through it, so I don't know if this change is the source of the issue, but I'm guessing it is.

Reply:
Thanks! I'll take a closer look at that!

Reply:
Resolved. Added `&` to make sure it wasnt calling functions if the date was null